test_scripts/API/VehicleData/Versioning/Version_5_0_and_higher/001_GetVehicleData_TwoParams_OneOfVersion_5_0_Another_HasNoVersion.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2997
+--
+-- Description:
+--   SDL sends GetVehicleData request with fuelRange and tirePressure to HMI 
+--   when both params are valid by mobile API
+--
+-- Expected behavior:
+--   SDL passes request successfully
+--
+-- Steps to reproduce :
+--   1. Mobile application sends valid GetVehicleData request to SDL with params:
+--      1) fuelRange, that is available from syncMsgVersion 5.0 only;
+--      2) tirePressure, that has no versioning
+--
+--   2.SDL must:
+--     1) Transfer GetVehicleData request to HMI 
+--     2) After successful response from HMI response SUCCESS to Mobile App, 
+--        with parameter value received from HMI
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/Versioning/common')
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Local Variables ]]
+local rpc = {
+  name = "GetVehicleData",
+  params = {
+    fuelRange = true, -- available from 5.0
+    tirePressure = true
+  }
+}
+
+local vehicleDataValues = {
+  tirePressure = {
+    leftFront = {
+      status = "NORMAL"
+    },
+    rightFront = {
+      status = "NORMAL"
+    }
+  },
+  fuelRange = {
+    { 
+      type = "GASOLINE", 
+      range = 400.00 
+    }
+  }
+}
+
+--[[ Local Functions ]]
+local function processRPCSuccess()
```

Comment:
@YarikMamykin Please move functions which are used in more than one script to common module.